__tests__/components/sections/communities/overview/scoreboard/Filter.test.tsx

```diff
@@ -0,0 +1,57 @@
+import Filters, { filterOptions, sortingOptions } from "@/components/sections/communities/overview/scoreboard/Filter";
+import "@testing-library/jest-dom";
+import { fireEvent, screen } from "@testing-library/react";
+import { useRouter } from "next/router";
+import { renderWithRedux } from "@__mocks__/renderWithRedux";
+
+jest.mock("next/router", () => ({
+  useRouter: jest.fn(),
+}));
+
+describe("Filter", () => {
+  const mockRouter = {
+    query: { slug: "test-slug" },
+    locale: "en",
+    push: jest.fn(),
+  };
+
+  beforeEach(() => {
+    (useRouter as jest.Mock).mockReturnValue(mockRouter);
+  });
```

Comment:
There is no need for this if you add the content of mockRouter in the jest.mock on top